src/services/railcontent.js

```diff
@@ -236,10 +237,10 @@ export async function fetchContentPageUserData(contentId) {
     }
 }
 
-export async function fetchUserPermissions() {
-    let url = `/content/user_data_permissions`;
+export async function fetchUserPermissionsData() {
+    let url = `/content/user/permissions`;
```

Comment:
Is there a MWP PR for this that I'm missing? I couldn't find this route.

Reply:
I just pushed it because it was very straightforward.
https://web-staging-one.musora.com/content/user/permissions